src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static addAlive(animal) {
+    this.alive.push(animal);
+  }
+
+  static removeAlive(animal) {
+    Animal.alive = this.alive.filter(item => item.health > 0);
+  }
```

Comment:
```suggestion
  static removeAlive() {
    Animal.alive = this.alive.filter(animal => animal.health > 0);
  }
```
u dont use animal parameter + proper naming